test/development/app-dir/owner-stack/owner-stack.test.ts

```diff
@@ -74,8 +77,7 @@ describe('app-dir - owner-stack', () => {
      "Error: browser error
      at useThrowError 
      at useErrorHook 
-     at Page 
-     at ClientPageRoot"
+     at Page"
```

Comment:
This is a fix. We used to always append to Owner Stack in the console. This would be redundant in Chrome or when React DevTools is installed. Chrome already has `createTask` so you get the Owner Stack from there. RDT automatically appends the Owner Stack if `createTask` is not available and the user hasn't disabled automatically appending stacks.